src/routes/users_test.ts

```diff
@@ -0,0 +1,102 @@
+import { assert } from "https://deno.land/std@0.160.0/_util/assert.ts";
```

Comment:
Pekne, jako tresnicku na dortu: Oznacil bych soubor (libovolnym zpusobem) tj komentar/prejmenovani souboru, aby bylo udelano rozliseni, ze jsou to integracni testy a ze vyzaduji urcite nastaveni serveru (viz fetch z localhost:5000).

Zbytek k tomuto souboru jsem napsal na discordu:)